docs/book/src/user/quick-start.md

```diff
@@ -1184,10 +1203,24 @@ Please visit the [KubeKey provider] for more information.
 {{#/tab }}
 {{#tab KubeVirt}}
 
-```bash
-export CAPK_GUEST_K8S_VERSION="v1.23.10"
-export CRI_PATH="/var/run/containerd/containerd.sock"
-export NODE_VM_IMAGE_TEMPLATE="quay.io/capk/ubuntu-2004-container-disk:${CAPK_GUEST_K8S_VERSION}"
+The following VM imagees are available:
```

Comment:
I think it's a good idea to help the user figure out which image to use, but we probably don't want to maintain the list here. How about a link instead?

Reply:
I don't think we should aim to be exhaustive in a quickstart tutorial. IMO it's totally fine to put a link e.g. to https://quay.io/repository/capk/ubuntu-2204-container-disk?tab=tags&tag=latest and maybe a note pointing to a guide for building custom container disks in case other versions or distros are desired.